doc/api/module.md

```diff
@@ -399,6 +399,18 @@ the [`NODE_COMPILE_CACHE=dir`][] environment variable if it's set, or defaults
 to `path.join(os.tmpdir(), 'node-compile-cache')` otherwise. To locate the compile cache
 directory used by a running Node.js instance, use [`module.getCompileCacheDir()`][].
 
+When portable mode is enabled, cache keys use paths relative to the compile cache directory.
+This allows the cache to be reused after moving the project across directories etc.
+
+It can be enabled via:
+
+```js
+module.enableCompileCache({ path: '...', portable: true });
+```
```

Comment:
It's likely worth expanding the example here a bit more to illustrate the difference between `false` and `true`. As it is, someone who isn't that familiar with how this is implemented likely wouldn't be able to determine exactly when to use this.

Reply:
Yes, much better :-)